pkg/install/condition.go

```diff
@@ -35,16 +28,11 @@ func (i *Installer) operatorConsoleExists() (bool, error) {
 }
 
 func (i *Installer) operatorConsoleReady() (bool, error) {
-	operatorConfig, err := i.operatorcli.OperatorV1().Consoles().Get(consoleapi.ConfigResourceName, metav1.GetOptions{})
-	if err == nil && operatorConfig.Status.ObservedGeneration == operatorConfig.Generation {
-		for _, cond := range operatorConfig.Status.Conditions {
-			if cond.Type == "Deployment"+operatorv1.OperatorStatusTypeAvailable &&
-				cond.Status == operatorv1.ConditionTrue {
-				return true, nil
-			}
-		}
+	consoleOperator, err := i.configcli.ConfigV1().ClusterOperators().Get("kube-apiserver", metav1.GetOptions{})
```

Comment:
kube-apiserver here is a copy/paste error?

Reply:
copy/paste - corrected to console.